test/getJwkDiscovery.spec.js

```diff
@@ -36,7 +39,11 @@ t.test('rejects if the request fails', async t => {
   const [{ alg, kid }] = jwks.keys
   const getJwks = buildGetJwks({ providerDiscovery: true })
 
-  const expectedError = new Error('Internal Server Error')
+  const expectedError = {
```

Comment:
why do you need to create the error in this way rather than creating an instance of the error?

Reply:
That's the issue I hinted at. I did not see a way to create an instance of that error that passes the built-in matcher because that would require me to provide the correct `response` object:
```typescript
const expectedError = new GetJwksError(errorCode.JWKS_REQUEST_FAILED, {
  response: new Response({}, { timeout: 5000 }),
  body: { msg: 'boom' },
})
```
...and that just didn't feel right because it is then asserting some internals that should not be relevant.

In other test frameworks I would do something like this:
```typescript
const expectedError = new GetJwksError(errorCode.JWKS_REQUEST_FAILED, {
  response: expect.anyt(Response),
  body: { msg: 'boom' },
})
```
but I did not see a way to do this with `tap`.